### PR TITLE
Run formatter on lib/test/mix/tasks/profile.fprof_test.exs

### PR DESCRIPTION
**Pull request by AshayTikekar**
Run formatter on lib/test/mix/tasks/profile.fprof_test.exs

### CODE REVIEW

File: lib/mix/test/mix/tasks/profile.fprof_test.exs [hunk 1]
```diff
@@ -11,38 +11,44 @@ defmodule Mix.Tasks.Profile.FprofTest do
...
   test "expands processes", context do
     in_tmp context.test, fn ->
-      output = capture_io(fn ->
-        Fprof.run(["-e", "spawn(fn -> Process.sleep(:infinity) end); Enum.each(1..5, fn(_) -> MapSet.new end)", "--details"])
-      end)
+      output =
+        capture_io(fn ->
+          Fprof.run([
+            "-e",
+            "spawn(fn -> Process.sleep(:infinity) end); Enum.each(1..5, fn(_) -> MapSet.new end)",
```
Comment (josevalim) [inline lib/mix/test/mix/tasks/profile.fprof_test.exs]: Can you please extract this out so everything is a single line?

```
expr = "spawn(fn -> Process.sleep(:infinity) end); Enum.each(1..5, fn(_) -> MapSet.new end)"
output = capture_io(fn -> Fprof.run(["-e", expr, "--details"]) end)
assert output =~ ...
```

File: lib/mix/test/mix/tasks/profile.fprof_test.exs [hunk 2]
```diff
@@ -53,45 +59,50 @@ defmodule Mix.Tasks.Profile.FprofTest do
...
   test "errors on missing files", context do
     in_tmp context.test, fn ->
-      assert_raise Mix.Error, "No files matched pattern \"non-existent\" given to --require", fn ->
-        capture_io(fn -> Fprof.run ["-r", "non-existent"] end)
-      end
-
-      assert_raise Mix.Error, "No files matched pattern \"non-existent\" given to --require", fn ->
-        capture_io(fn -> Fprof.run ["-pr", "non-existent"] end)
-      end
+      assert_raise Mix.Error,
```
Comment (josevalim) [inline lib/mix/test/mix/tasks/profile.fprof_test.exs]: Please format this and below as in example #3 here: https://github.com/elixir-lang/elixir/issues/6643

Comment (AshayTikekar) [reply lib/mix/test/mix/tasks/profile.fprof_test.exs]: sure....

File: lib/mix/test/mix/tasks/profile.fprof_test.exs (hunk 1 above)
Comment (AshayTikekar) [reply lib/mix/test/mix/tasks/profile.fprof_test.exs]: yes i will do that

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**